src/common/outputService/aorOutputService.ts

```diff
@@ -76,8 +76,7 @@ export abstract class AbstractAorOutputService<T extends AorOutputFlags>
       } else {
         console.log(red(aor.sf_devops__Message__c));
       }
-    } else {
-      // aor.sf_devops__Status__c == undefined || AsyncOperationStatus.InProgress
+    } else if (aor.sf_devops__Message__c) {
```

Comment:
I see line 77 and 68 that also print sf_devops__Message__C without checking it is null. Is that okey?

Reply:
When we update the Status field to Complete or to Error we also update the Message field, in the same transaction. So I think for those to cases the Message field will always have a value.
When the Status is In Progress it is posible that Message is null because we can be updating other fields and not Message.

Do you think might be a scenario where the Status is Complete|Error and Message is null? If so I can add the check there too.

Reply:
Yes, completed for sure has. And error we have logic in place to either log the message, if not the exception message, if not the exception so that should cover it.

If we see something we can revisit this.